R/regression_interactive.R

```diff
@@ -1,16 +1,36 @@
-#' interactive_regression() runs a regression simulation.
-#' Click on the plotting area to add points and see a corresponding regression line
-#'  (hitting ESC will stop the simulation). 
+#' compstatslib interactive_regression() function
+#' 
+#' Interactive visualization function that lets you point-and-click to add data points, while it automatically plots and updates a regression line and associated statistics.
+#'   
+#' @param points A dataframe containing coordinates for estimating the regression.
```

Comment:
An optional dataframe of (x,y) points to plot and estimate the regression. If no points are are provided, the user is free to click and create points.